### ISSUE TITLE
Error processing tar file(exit status 1): unexpected EOF

### ISSUE THREAD

**Issue by pwasem** [tobiashofmann/sap-nw-abap-docker#5]
Running `docker build -t nwabap .` with [NW752](https://www.sap.com/developer/trials-downloads/additional-downloads/sap-netweaver-as-abap-developer-edition-sp01-7-52-15510.html)
ends with this error message: 
`Error processing tar file(exit status 1): unexpected EOF`

So building the container fails.

```
[...]

Checking syb Database
Database is running
-------------------------------------------
Starting Startup Agent sapstartsrv
OK
Instance Service on host 76b9bf849409 started
-------------------------------------------
starting SAP Instance ASCS01
Startup-Log is written to /home/npladm/startsap_ASCS01.log
-------------------------------------------
/usr/sap/NPL/ASCS01/exe/sapcontrol -prot NI_HTTP -nr 01 -function Start
Instance on host 76b9bf849409 started
Starting Startup Agent sapstartsrv
OK
Instance Service on host 76b9bf849409 started
-------------------------------------------
starting SAP Instance D00
Startup-Log is written to /home/npladm/startsap_D00.log
-------------------------------------------
/usr/sap/NPL/D00/exe/sapcontrol -prot NI_HTTP -nr 00 -function Start
Instance on host 76b9bf849409 started
Installation of NPL successful
 
Error processing tar file(exit status 1): unexpected EOF
```

**Comment by tobiashofmann** [comment]
Hi @pwasem 
not sure what the EOF causes. NPL is installed and started successfully. 

What comes to my mind that may cause an error:

- The docker image is large enough? 
- Does it happen always or were you able to create the container successfully? 
- When you take the image and start a container from it, does this work?

**Comment by pwasem** [comment]
Hi @tobiashofmann ,
- Docker disc image size 256GB, CPUs: 4,  Memory 6GB, Swap: 1GB (additional steps have been performed)
- It happens every time I try to build the image
- The image gets not built successfully, so I am not able to start the container

Maybe the zip file is corrupt? Did you get a succesfull for NW752 yet?

Thank you in advance!

**Comment by tobiashofmann** [comment]
HI @pwasem 

you can download the files from SAP again and see if it works, but I do not think the files are corrupt. You get up to the step when the installer starts NPL. The installation worked. 

What you can try is to not run the run.sh script from the Dockerfile (comment it out). Put the files in the image, create a container and start the installation manually and see what happens.

**Comment by wozjac** [comment]
Hi,
thanks for this project.
I successfully installed the server, startsap ALL fires up the server, but still, I don't know how to perform the rest of the setup under Windows 10.

I have inspected the docker container via "docker inspect nwabap752", got the port for the SAPGUI.

"Ports": {
                "3200/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "32771"
                    }
                ],

Then I follow the video: https://youtu.be/H0GEg8r7P48?t=653

So I put the connection string conn=/H/localhost/S/32771 to SAPGUI Java
but what I get is:
Error: connection to partner '127.0.0.1:32771' broken

Any clues?

Regards,
Jacek
